assets/scss/_base-layout-and-modules.scss

```
@@ -399,10 +399,16 @@ blockquote::after {
 }
 
 // Callout warning — functional alert box
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)